components/script/dom/dedicatedworkerglobalscope.rs

```diff
@@ -608,13 +609,14 @@ impl DedicatedWorkerGlobalScope {
         cx: SafeJSContext,
         message: HandleValue,
         transfer: CustomAutoRooterGuard<Vec<*mut JSObject>>,
+        can_gc: CanGc,
     ) -> ErrorResult {
         let data = structuredclone::write(cx, message, Some(transfer))?;
         let worker = self.worker.borrow().as_ref().unwrap().clone();
         let global_scope = self.upcast::<GlobalScope>();
         let pipeline_id = global_scope.pipeline_id();
         let task = Box::new(task!(post_worker_message: move || {
-            Worker::handle_message(worker, data);
+            Worker::handle_message(worker, data, can_gc);
```

Comment:
This is within a task! macro, so we should not propagate it.

Reply:
i shall fix that